api/hypershift/v1beta1/hostedcluster_conditions.go

```diff
@@ -251,6 +258,12 @@ const (
 	RecoveryFinishedReason = "RecoveryFinished"
 
 	CloudResourcesCleanupSkippedReason = "CloudResourcesCleanupSkipped"
+
+	ControlPlaneToDataPlaneReasonNoKonnectivityAgentPodsFound = "NoKonnectivityAgentPodsFound"
```

Comment:
```suggestion
	KonnectivityAgentPodsNotFound = "KonnectivityAgentPodsNotFound"
```

Reply:
The reason is expressed like that 'cause the connection issue  may come from multiple problems. 
No nodes (in that case is Unknown)
No Pods or error returning Logs. the idea was to give customer some info about the source of problem